Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:10-alpine
 
-RUN yarn install -g create-react-app firebase-tools
+RUN yarn global add create-react-app firebase-tools
```

Comment:
Why ?

Reply:
Because it is deprecated, i did a docker-compose up -d --build and had an error which blocked the launch of containers.

Reply:
Ok thx